L1Trigger/Phase2L1ParticleFlow/python/mlAssociation_cfi.py

```diff
@@ -0,0 +1,12 @@
+import FWCore.ParameterSet.Config as cms
+from L1Trigger.VertexFinder.l1tVertexProducer_cfi import l1tVertexProducer
+
+# NNVertex Association Variables
+NNVtxAssociationPSet = cms.PSet(
+    associationThreshold = cms.double(0.1), #Association Network threshold for PV tracks
+    # associationGraph = cms.string("L1Trigger/L1TTrackMatch/data/NNVtx_AssociationModelGraph.pb"),
+    associationGraph = cms.string("L1Trigger/L1TTrackMatch/data/associationModelgraph.pb"),
```

Comment:
Have a temporary network name left in by accident; will revert after the code review

Reply:
fixed